### DISCUSSION TITLE
Why was GenerateLombokConfig deprecated?

### DISCUSSION THREAD

**Discussion by Pytry** [freefair/gradle-plugins#390] [Q&A] [answered]
I'm updating to version "6.2.0". My goal is to completely disable the generation of lombok.config files. Previously I could do that my disabling the task, but it seems the task was removed or renamed, and the GenerateLombokConfig class was deprecated.

How should we accomplish this now?

**Comment by larsgrefer** [comment, marked as answer]
Since Version 6.x the plugin does not generate `lombok.config` files anymore.
That is why the corresponding Task class was removed.

More information about the rationale can be found here: https://github.com/freefair/gradle-plugins/issues/379#issuecomment-900638937

**Comment by Pytry** [comment]
That makes sense. It always annoyed me that it would overwrite my config files if I didn’t disable that task.

Thanks for answering:)
